library/build.gradle

```diff
@@ -35,6 +35,7 @@ android {
 
 dependencies {
     implementation "androidx.recyclerview:recyclerview:1.1.0"
+    implementation "androidx.recyclerview:recyclerview-selection:1.1.0-rc01"
```

Comment:
this should be a `compileOnly` dependency as we don't want to force it on people.

Reply:
I fixed the code